compiler/aarch64/codegen/OMRRealRegister.hpp

```diff
@@ -47,7 +47,11 @@ namespace ARM64
 
 class OMR_EXTENSIBLE RealRegister : public OMR::RealRegister
    {
+<<<<<<< HEAD
 protected:
+=======
+   protected:
+>>>>>>> 2160301... WIP: Initial version of RealRegister-related files for aarch64
```

Comment:
I think you have a few git artifacts left over in this commit.

Reply:
Thank you for pointing this out.  I made a mistake in rebase operation.